pages/common/docker-network.md

```diff
@@ -0,0 +1,32 @@
+# docker network
+
+> Create and manage docker network.
+> More information: <https://docs.docker.com/engine/reference/commandline/network/>.
+
+- List all available and configured networks on docker daemon:
+
+`docker network ls`
+
+- Create user defined network:
+
+`docker network create --driver {{driver_name}} {{network_name}}`
+
+- Display detailed information on one or more networks:
+
+`docker network inspect {{network_name}}`
+
+- Connect a container to a network:
+
+`docker network connect {{network_name}} {{container_name/id}}`
+
+- Disconnect a container from a network:
```

Comment:
```suggestion
- Disconnect a container from a network using a name or ID:
```

Reply:
@navarroaxel, I guess `using a name or ID` is not required as it is a default understanding for each command. but, if you require it, we need to add it after each command information as each command has operation on network.